sdk/provisioning/Directory.Build.props

```diff
@@ -14,22 +14,8 @@
     <Nullable>enable</Nullable>
   </PropertyGroup>
 
-  <ItemGroup Condition="'$(AssemblyName)' != 'Azure.Provisioning' And '$(AssemblyName)' != 'Generator'">
-    <ProjectReference Include="../../Azure.Provisioning/src/Azure.Provisioning.csproj" />
-  </ItemGroup>
```

Comment:
the include is moved to Directory.Build.Common.props.
The version is in `Package.Data.props`.